src/mock/task.js

```diff
@@ -0,0 +1,68 @@
+import {getRandomInteger} from "../utils.js";
+import {getRandomColor} from "../utils.js";
+
+const generateDescription = () => {
+  const descriptions = [
+    `Изучить теорию`,
+    `Сделать домашку`,
+    `Пройти интенсив на соточку`
+  ];
+
+  const randomIndex = getRandomInteger(0, descriptions.length - 1);
+
+  return descriptions[randomIndex];
+};
+
+const generateDate = () => {
+  const isDate = Boolean(getRandomInteger(0, 1));
+
+  if (!isDate) {
+    return null;
+  }
+
+  const maxDaysGap = 7;
+  const daysGap = getRandomInteger(-maxDaysGap, maxDaysGap);
+  const currentDate = new Date();
+
+  currentDate.setHours(23, 59, 59, 999);
+
+  currentDate.setDate(currentDate.getDate() + daysGap);
+
+  return new Date(currentDate);
+};
+
+const generateRepeating = () => {
+  return {
+    mo: false,
+    tu: false,
+    we: Boolean(getRandomInteger(0, 1)),
```

Comment:
Тут все правильно, но, раз мы уже взялись писать в крутом синтаксисе, будет сложно остановиться :)
Если ты хочешь преобразовать что-то в буль, есть вариант короче.
```suggestion
    we: !!getRandomInteger(0, 1),
```
Оператор `!` это оператор `НЕ`, применив его дважды, приведем переменную к богоугодному булю.

Reply:
Дальше в коде есть такие штуки, можешь и там заменить)